CacheWarmer/AspectCacheWarmer.php

```diff
@@ -94,7 +93,7 @@ public function warmUp($cacheDir)
                     "/resource=" . $realPath
                 );
             } catch (\Exception $e) {
-                $errors[$realPath] = $e;
+                /* noop */
```

Comment:
Yes, agree, just dead code indeed.

For cache warmer we shouldn't report any errors, but what can be done is adding case for PHP7 errors: `catch (\Throwable $e)`. Can be added later.